### ISSUE TITLE
Setup - Need S3 object read permission

### ISSUE THREAD

**Issue by luckyvaliin** [qubole/spark-on-lambda#1]
As per instructions, I was trying to copy the file and errors out.

aws s3 cp s3://public-qubole/lambda/spark-2.1.0-bin-spark-lambda-2.1.0.tgz s3://myBucket-xxx/spark-on-lambda/spark-2.1.0-bin-spark-lambda-2.1.0.tgz --acl bucket-owner-full-control

Error:
fatal error: An error occurred (403) when calling the HeadObject operation: Forbidden

Could you provide appropriate access for the users to access the neccesary files.

Thanks
Vali

**Comment by venkata91** [comment]
Can you try running `aws s3 ls s3://public-qubole/lambda/spark-2.1.0-bin-spark-lambda-2.1.0.tgz`? I tried with a different set of keys and the copy worked.

**Comment by luckyvaliin** [comment]
Yes, certainly i am able to list the object and infact many other objects
under that hierarchy. But the copy doesn't work. I checked with AWS support
and they confirmed that the owner object permission is not properly set for
it to be copied. Let me know if you have any questions.

Thanks
Mahaboob Vali Shaik

**Comment by venkata91** [comment]
Alright, that was an issue from our side, the keys owner didn't allow public to do GET operation. I think now it should be fine, we changed the owner permissions. Can you please try again and let us know if that fixes the issue?
